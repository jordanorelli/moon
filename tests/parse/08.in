dur_1: 30s
dur_2: 300ms
dur_3: -1.5h
dur_4: 2h45m
dur_5: 9ns
dur_6: 18us
dur_7: 45Âµs
dur_8: 900ms
dur_9: 275s
dur_10: 89m
dur_11: 927h
