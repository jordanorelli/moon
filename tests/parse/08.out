root:
  assign:
    name:
      dur_1
    value:
      dur:
        30s
  assign:
    name:
      dur_2
    value:
      dur:
        300ms
  assign:
    name:
      dur_3
    value:
      dur:
        -1h30m0s
  assign:
    name:
      dur_4
    value:
      dur:
        2h45m0s
  assign:
    name:
      dur_5
    value:
      dur:
        9ns
  assign:
    name:
      dur_6
    value:
      dur:
        18us
  assign:
    name:
      dur_7
    value:
      dur:
        45us
  assign:
    name:
      dur_8
    value:
      dur:
        900ms
  assign:
    name:
      dur_9
    value:
      dur:
        4m35s
  assign:
    name:
      dur_10
    value:
      dur:
        1h29m0s
  assign:
    name:
      dur_11
    value:
      dur:
        927h0m0s
